src/App.js

```diff
@@ -44,28 +42,25 @@ function App() {
 }
 
```

Comment:
when I run this to my local, it has a warning to put a span, role, and aria-label. You can add it to compiled it successfully!